File: exercicios/codigosEmOutrasLinguagens/EX10REC/EX10RECv2.py
def exibir_invertido(n):
    if n == 0:
        return
    
    numero = int(input())
    exibir_invertido(n - 1)
    print(numero)

def main():
    n = int(input("Quantos números você deseja informar? "))
    
    print("Digite os números e veja-os na ordem inversa:")
    exibir_invertido(n)

if __name__ == "__main__":
    main()

File: exercicios/codigosEmOutrasLinguagens/EX10REC/EX10RECv2.c
#include <stdio.h>

void exibir_invertido(int n) {
    if (n == 0) {
        return;
    }

    int numero;
    scanf("%d", &numero);
    exibir_invertido(n - 1);
    printf("%d\n", numero);
}

int main() {
    int n;

    printf("Quantos números você deseja informar? ");
    scanf("%d", &n);

    printf("Digite os números e veja-os na ordem inversa:\n");
    exibir_invertido(n);

    return 0;
}